src/controllers/issuer.ts

```diff
@@ -226,39 +225,54 @@ export class IssuerController {
 		}
 
 		const {
-			methodSpecificIdAlgo,
+			identifierFormatType,
 			network,
 			verificationMethodType,
-			assertionMethod = true,
-			serviceEndpoint,
+			service,
+			key,
+			options,
 		} = request.body;
 		let didDocument: DIDDocument;
 		try {
 			if (request.body.didDocument) {
```

Comment:
Same logic applies based on the above type cast.

- Define didDocument as part of the request.body object destructuring.
- Switch to didDocument declaration here.

Reply:
I cannot do that because in this case if I add `didDocument` to the list above then `didDocument` will be `const` since in my case `didDocument` should not be `const` because I try to change it value in else statement.